cmd/ims/config.go

```diff
@@ -0,0 +1,8 @@
+package main
+
+import "github.com/omc-college/management-system/pkg/db"
+
+type Config struct {
+	DBConnection 	db.ConnectionConfig 	`mapstructure:"dbconnection"`
+	WebAPIAddress 	string 			  		`mapstructure:"webapi_address"`
```

Comment:
it has excessive spaces between tag and field type

Reply:
It's a bug of GitHub. There is not such a big space

Reply:
That's how it looks like on my desktop

![2020-06-22 (16)](https://user-images.githubusercontent.com/61523411/85290504-ca994c80-b4a1-11ea-9932-12ed419ef003.png)